src/app/component/eco-news/components/eco-news-detail/eco-news-detail.component.scss

```diff
@@ -64,6 +68,11 @@ $main-font: 'Open Sans', sans-serif;
   text-align: center;
 }
 
+.edit-news:hover {
+  border-color: #056b33;
```

Comment:
I'm not sure but, maybe better to use variable

Reply:
It should be done for the whole file, but it is not the aim of this PR.